core/src/test/scala/com/microsoft/azure/synapse/ml/logging/HostEndpointUtilsTests.scala

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.azure.synapse.ml.logging
+
+import com.microsoft.azure.synapse.ml.core.test.base.TestBase
+import com.microsoft.azure.synapse.ml.logging.Usage.FabricConstants.{WorkloadEndpointAdmin}
+import com.microsoft.azure.synapse.ml.logging.Usage.HostEndpointUtils._
+
+class UsageUtilsTests extends TestBase {
```

Comment:
is this used in multiple places? if not just add these into the test where its used

Reply:
Let's have it in the header section for now as with the only one test that's within this file the potential to conflict and misuse is zero. Also, I might add tests in the future, that might share the imports.

Reply:
Just mix these methods into your test class, we can re-pull them out later. The goal is to make this as simple as possible

Reply:
This still applies, trying to cut down on the dangling small objects in this PR